coralproject/talk
# Fixes for Spam Plugin

## What does this PR do?

Spam comment notice only shows up when it is detected.

## How do I test this PR?

1. Enable the Akismet Spam plugin
2. Create a comment (that doesn't contain spam)
3. Notice the lack of spam markers in the moderation panel
